test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt

```diff
@@ -426,12 +426,21 @@ sun/security/krb5/ktab/FileKeyTab.java https://github.com/ibmruntimes/openj9-ope
 sun/security/krb5/ktab/KeyTabIndex.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 sun/security/krb5/runNameEquals.sh https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 sun/security/krb5/auto/Cleaners.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x
-java/lang/ProcessBuilder/JspawnhelperWarnings.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/ProcessBuilder/JspawnhelperWarnings.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 java/lang/String/CompactString/NegativeSize.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
-java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java#id0 https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java#id0 https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 java/security/Security/ConfigFileTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x
 javax/crypto/CipherSpi/ResetByteBuffer.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-s390x
 java/security/Security/ConfigFileTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/System/LoggerFinder/SignedLoggerFinderTest/SignedLoggerFinderTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,aix-all,linux-ppc64le,linux-s390x
```

Comment:
Is this the same as [this issue](https://github.com/eclipse-openj9/openj9/issues/19825)? If yes, it was fixed [here](https://github.com/ibmruntimes/openj9-openjdk-jdk22/pull/77).

Reply:
@JasonFengJ9 could you please confirm if this fix is included in the release build?

Reply:
As per https://github.com/eclipse-openj9/openj9/issues/19825#issuecomment-2232068677, all fixes have been merged and included in the 0.46.0 release branches.

Reply:
updated

Reply:
Since it has already been addressed, no need to update the excluded list for jdk11 and jdk17.